.clang-tidy

```diff
@@ -1,32 +1,35 @@
 ---
-# NOTE there must be no spaces before the '-', so put the comma first.
+# NOTE there must be no spaces before the '-' and check name.
+# If you edit this listplease virify list of enabled check with
+#   clang-tidy --list-checks
```

Comment:
To enforce this, we should add this check to CircleCI [here](https://github.com/pytorch/audio/tree/master/.circleci/unittest/linux/scripts)